### ISSUE TITLE
Can`t get gradient from MHR model

### ISSUE THREAD

**Issue by R132** [facebookresearch/MHR#61]
After 
        vertices, skeleton_state = mhr_model(
            identity_coeffs,
            model_parameters,
            face_expr_coeffs
        )

vertices.requires_grad  is False.  Any way to enable gradient ?

**Comment by jeongseok-meta** [comment]
MHR is differentiable with respect to its input tensors. `vertices.requires_grad` will remain `False` when all inputs use PyTorch's default `requires_grad=False`. Also make sure the forward pass is not inside `torch.no_grad()`; the calls in `demo.py` use that context because the demo performs inference only.

I verified the following against both the native `MHR` model and the shipped TorchScript model:

```python
identity_coeffs.requires_grad_()
model_parameters.requires_grad_()
face_expr_coeffs.requires_grad_()

vertices, skeleton_state = mhr_model(
    identity_coeffs,
    model_parameters,
    face_expr_coeffs,
)

loss = vertices.square().mean()
loss.backward()
```

With this code, `vertices.requires_grad` is `True`, and all three inputs receive finite gradients. Could you try it outside any `torch.no_grad()` context and confirm whether it resolves the issue in your setup?

I opened [#81](https://github.com/facebookresearch/MHR/pull/81) to document this behavior and add regression coverage. I'll keep this issue open pending your confirmation.
